src/app/authorisation/authorisation.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface Authorisation {
```

Comment:
Why do you need this interface?

Reply:
13. Create a user entity typescript interface. User entity has the next fields:
Id
First Name
Last Name

It has been in task